.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: python
+python:
+  # We don't actually use the Travis Python, but this keeps it organized.
+  - "2.7"
+  - "3.4"
+before_install:
+  - sudo apt-get update -qq
+install:
+  # You may want to periodically update this, although the conda update
+  # conda line below will keep everything up-to-date.  We do this
+  # conditionally because it saves us some downloading if the version is
+  # the same.
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
+      wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh;
+    else
+      wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+    fi
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - hash -r
+  - conda config --set always_yes yes --set changeps1 no 
+  - conda config --add channels pyne
+  - conda config --add channels cyclus
+  - conda update -q conda
+  - conda install conda-build jinja2 setuptools binstar patchelf nose
```

Comment:
Attempted to launch Travis in my repo and this is the error: https://travis-ci.org/opotowsky/cymetric

I'm posting here because I'm wondering if glib needs to be included on this line (@gidden @scopatz)